services/web/client/source/class/osparc/component/widget/newStudy/Basic.js

```diff
@@ -0,0 +1,171 @@
+/* ************************************************************************
+
+   osparc - the simcore frontend
+
+   https://osparc.io
+
+   Copyright:
+     2018 IT'IS Foundation, https://itis.swiss
+
+   License:
+     MIT: https://opensource.org/licenses/MIT
+
+   Authors:
+     * Odei Maiz (odeimaiz)
+
+************************************************************************ */
+
+/**
+ * Widget that provides the form for creating a new study
+ *
+ * After doing some validation the following data event is fired:
+ * <pre class='javascript'>
+ *   {
+ *     prjTitle: title,
+ *     prjDescription: desc,
+ *     prjThumbnail: thumb,
+ *     prjWorkbench: wb  // (optional)
+ *   };
+ * </pre>
+ *
+ * *Example*
+ *
+ * Here is a little example of how to use the widget.
+ *
+ * <pre class='javascript'>
+ *   let newStudyBasic = new osparc.component.widget.newStudy.Basic();
+ *   this.getRoot().add(newStudyBasic);
+ * </pre>
+ */
+
+qx.Class.define("osparc.component.widget.newStudy.Basic", {
+  extend: qx.ui.core.Widget,
+
+  construct: function(template=null, withPipeline=false) {
+    this.base(arguments);
+
+    this._setLayout(new qx.ui.layout.VBox(5));
+
+    this.__createForm(template, withPipeline);
+  },
+
+  events: {
+    "createStudy": "qx.event.type.Data"
+  },
+
+  members: {
+    _createChildControlImpl: function(id) {
+      let control;
+      switch (id) {
+        case "studyTitle":
+          control = new qx.ui.form.TextField().set({
+            placeholder: this.tr("Study Title")
+          });
+          osparc.utils.Utils.setIdToWidget(control, "newStudyTitleFld");
+          this.addListener("appear", () => {
+            control.activate();
+            control.focus();
+          });
+          this._add(control);
+          break;
+        case "studyDescription":
```

Comment:
I think the Qooxdoo convention for child names is all lowercase with dashes: ``study-description``